vllm/worker/habana_model_runner.py

```diff
@@ -1350,9 +1350,9 @@ def warmup_scenario(self,
         times = 3 if use_graphs or is_pt_profiler_run else 1
         if self.lora_config and not is_lora_profile_run:
             lora_mapping = LoRAMapping(
-                [0] * batch_size * seq_len,
-                [0] * batch_size * seq_len,
-            )
+                **dict(index_mapping=[0] * batch_size * seq_len,
```

Comment:
how does introducing dict here affect memory footprint? since we dont have control on when its garbage collected and size grows with batch_size*seq_len

Reply:
@michalkuligowski Nothing has changed from existing code, we are still populating the same dataclass (LoRAMapping) as before,  **dict simply is to use key, value notation initialization of class members.  Regarding memory footprint, this dataclass needs to be available all the time if LoRA is enabled, it is a must have for LoRA operation. Again nothing specific to Gaudi/HPU, this is the way LoRA feature is implemented in vLLM.